release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -42,6 +42,31 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### Language
 
+##### Improvements in annotations support
+
+`mapping-constructor-expr` is now allowed be optional in annotations, if the annotation type requires a map.
```

Comment:
```suggestion
The `mapping-constructor-expr` is now allowed to be optional in annotations if the annotation type requires a map.
```

Reply:
What we want to say is: annotation value is now allowed to be omitted from an annotation, if the annotation value is empty mapping constructor.
@MaryamZi  please help us word this.

Reply:
Let's simply say that. :)

"Now, it is optional to have the `mapping-constructor-expr` in an annotation if the annotation value is an empty mapping constructor."

Reply:
Shall we say something like

```suggestion
If the type of the annotation is a mapping type for which an empty mapping constructor is valid, the mapping constructor expression is no longer mandatory in the annotation attachment.

The absence of the mapping constructor expression in such an annotation attachment is equivalent to specifying a mapping constructor expression with no fields.
```